packages/docs/src/pages/qwikcity/INDEX

```diff
@@ -0,0 +1,50 @@
+# Qwikcity Guide
+
+## Qwikcity
+
+- [Overview](overview.mdx)
+- [File Layout](file-layout.mdx)
+- [Static Assets](static-assets.mdx)
+
+## Routing
+
+- [Overview](routing/overview.mdx)
+- [Index](routing/default-index.mdx)
+- [Slugs](routing/slugs.mdx)
+- [Pathless Dirs](routing/pathless.mdx)
+
+## Layout
+
+- [Overview](layout/overview.mdx)
+- [Nested](layout/nested.mdx)
+- [Named](layout/named.mdx)
+
+## Menus
+
+- [Overview](menu/overview.mdx)
+- [File Layout](menu/file-definition.mdx)
+
+## Authoring Content
+
+- [Overview](content/overview.mdx)
+- [Components](content/component.mdx)
+- [MDX](content/mdx.mdx)
+
+## API Endpoints
+
+- [Overview](endpoint/overview.mdx)
+
+## useMethod() APIs
+
+- [useContentBreadcrumbs()](api/use-content-breadcrumbs.mdx)
+- [useContentHeadings()](api/use-content-headings.mdx)
+- [useContentMenu()](api/use-content-menu.mdx)
+- [useDocumentHead()](api/use-document-head.mdx)
+- [useLocation()](api/use-location.mdx)
+- [<HTML>](api/html.mdx)
+- [<Content>](api/content.mdx)
```

Comment:
`content.mdx` is missing.
Clicking the link breaks the page.

> http://localhost:3000/qwikcity/api/content

<img width="867" alt="CleanShot 2022-07-06 at 22 12 53@2x" src="https://user-images.githubusercontent.com/444278/177634920-5b02bfd3-3be6-464b-8f14-737d1938cb4f.png">

Reply:
work in progress, should be working now.